### ISSUE TITLE
[Question]: is `fabric` an unhappy name? 

### ISSUE THREAD

**Issue by pilasguru** [danielmiessler/Fabric#1112]
### What is your question?

```
$ brew info fabric
==> fabric: stable 3.2.2 (bottled), HEAD
Library and command-line tool for SSH
https://www.fabfile.org/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/f/fabric.rb
License: BSD-2-Clause
==> Dependencies
Build: rust ✘
Required: cryptography ✘, libsodium ✔, python@3.13 ✔
```

**Comment by BryanDollery** [comment]
There are so many software projects called Fabric that it's difficult to search for this project amongst all the noise. But, naming is hard, and unique naming is harder, so...

**Comment by CuriousJ415** [comment]
I agree with you @BryanDollery, it is hard to search for solutions or reference the software by the name "Fabric" - too many conflicting software titles or references to "Fabric" as a "meta-substrate" in software development.  Maybe FabricAI, FabricDM, FabbyDabby, Fabtastic, Fabulicious?

**Comment by onsomlem** [comment]
I think fabric is good if you search fabric GitHub with no other context this comes up so I think it is perfect.

**Comment by ksylvan** [comment]
The name (and community) has been established for a while now. We have no plans to change it. Thank you.
